test/verify/check-sosreport

```diff
@@ -35,20 +35,27 @@ class TestSOS(MachineCase):
         b = self.browser
         m = self.machine
 
-        # TODO - Sos 4 has moved config to /etc/sos/sos.conf and uses a different format.
-        # But we can't use that because of https://bugzilla.redhat.com/show_bug.cgi?id=1882015
-        self.restore_file("/etc/sos.conf")
-        m.write("/etc/sos.conf",
-                """
+        if m.image in ["centos-8-stream", "debian-stable"]: # Still has sos in version 3.x
```

Comment:
FTR, the refresh in https://github.com/cockpit-project/bots/pull/1441 happens to bring in sos 4 to centos-8-stream. But in the interest of proper transactions, let's land this first and then do a follow-up which bumps it for that image refresh.